tests/scripts/garp_service.py

```diff
@@ -37,7 +41,13 @@ def gen_garp_packets(self):
                                                 ip_snd=source_ip,
                                                 ip_tgt=source_ip, # Re-use server IP as target IP, since it is within the subnet of the VLAN IP
                                                 arp_op=2)
-            self.packets[intf_name] = garp_pkt
+
+            na_pkt = Ether(src=source_mac, dst=dut_mac) \
```

Comment:
is this for the untracked na coverage?

Reply:
Not specifically for the untracked NA coverage, this is to get the ToR to learn IPv6 neighbors, we can then set these the links for these neighbors to standby to make sure the ToR will still proxy ARP for them even though the route for the neighbor points to the tunnel